content/en/docs/reference/glossary/workload.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: Workload
-id: workload
-date: 2019-02-12
-full_link: /docs/concepts/workloads/
+title: Workloads
+id: workloads
+date: 2019-02-13
+full_link: https://kubernetes.io/docs/concepts/workloads/
```

Comment:
the full link should still be the relative path to the docs (in other words, revert this change). See other glossary examples.